client/a8c-for-agencies/components/content-sidebar/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Main from 'calypso/components/main';
+import SidebarNavigation from 'calypso/jetpack-cloud/sections/partner-portal/sidebar-navigation';
+
+import './style.scss';
+
+type Props = {
+	mainContent: React.ReactNode;
+	rightSidebar: React.ReactNode;
+};
+
+const ContentSidebar = ( { mainContent, rightSidebar }: Props ) => {
+	return (
+		<>
+			<SidebarNavigation />
+			<Main wideLayout className="content-sidebar">
```

Comment:
We can also remove `<Main />` and have a div with the same class name as this is already handled as a part of the Layout. 

```suggestion
			<div className="content-sidebar">
```